revolv/templates/project/project.html

```diff
@@ -17,53 +22,118 @@
 {% include "project/partials/payment_modal.html" %}
 {% include "project/partials/confirm_modal.html" %}
 {% include "project/partials/success_modal.html" with project=project %}
-<div class="row">
-    <div class="large-12 columns">
-        <h1><strong>{{ project.title }}</strong></h1>
-        {% if project.tagline %}
-            <h3>{{project.tagline}}</h3>
-        {% endif %}
-        <ul class="button-group nav-tab-button-group">
-            {% if project.is_active %}
-                {% if user.is_authenticated %}
-                    <li><a id="donate-button" class="small button success radius" href="#donate" data-reveal-id="donate-modal">Donate</a></li>
-                {% else %}
-                    <li><a id="donate-button" class="small button success radius" href="/signin/?next=/project/{{project.pk}}/&reason=donate#login">Donate</a></li>
-                {% endif %}
-            {% endif %}
-        </ul>
-        <h3>Funding Goal : {{ project.funding_goal }}</h3>
-        <h3>End Date : {{ project.end_date }}</h3>
-        <h5>Location : {{ project.location}}</h5>
-        <div id="map-canvas"></div>
-        <p>Organization : {{project.org_name}}</p>
-        <p>About {{project.org_name}} : {{project.org_about}}</p>
-        {% if project.cover_photo %}
-            <img src="{{project.cover_photo.url}}"></img>
-        {% endif %}
-        <p>Mission Statement : {{ project.mission_statement }}</p>
-        {% if project.video_url %}
-            <iframe width="420" height="315" id="project_video"></iframe>
-        {% endif %}
-        {% if project.categories|length > 0 %}
-            <h5>Categories</h5>
-            {% for category in project.categories %}
-                {{ category }}
-            {% endfor %}
-        {% endif %}
-        {% if project.donation_levels|length > 0 %}
-            <h5>Donation Levels</h5>
-            {% for donation in project.donation_levels %}
-                <p>Description: {{ donation.description }}, Amount: {{ donation.amount }}</p>
-            {% endfor %}
-        {% endif %}
-        {% if project.is_completed or project.is_active and project.post_funding_updates %}
-            <h5>Post Funding Updates</h5>
-            <p>{{ project.post_funding_updates}}</p>
-        {% endif %}
+
+<div class="row dashboard">
```

Comment:
If these are all absolutely positioned, I don't think there's any point in making them rows